scripts/fuzz_opt.py

```diff
@@ -805,6 +805,12 @@ class D8Liftoff(D8):
             def run(self, wasm):
                 return super(D8Liftoff, self).run(wasm, extra_d8_flags=V8_LIFTOFF_ARGS)
 
+        class D8Turboshaft(D8):
+            name = 'd8_turboshaft'
+
+            def run(self, wasm):
+                return super(D8Turboshaft, self).run(wasm, extra_d8_flags=['--no-liftoff', '--turboshaft-wasm', '--turboshaft-wasm-instruction-selection-staged'])
```

Comment:
What does `--turboshaft-wasm-instruction-selection-staged` mean?

Reply:
I think it means that it enables the "staged" (almost ready) part of wasm instruction selection. That's all I know, though - these are the flags that were suggested to me :smile: